python-lib/ontology_tagger.py

```diff
@@ -132,7 +137,11 @@ def _tokenize_keywords(
         if self.lemmatization:
             self.tokenizer._activate_components_to_lemmatize(language)
         keywords = [
-            lowercase_if(text=keyword, lowercase=self.normalize_case)
+            unicode_normalize_text(
+                text=lowercase_if(text=keyword, lowercase=self.normalize_case),
+                use_nfc=self._use_nfc,
+                normalize_diacritics=self.normalize_diacritics,
+            )
```

Comment:
DRY: Maybe you could place this in a separate fonction as it is also used lines 110 -> 113